ironfish/src/memPool/memPool.ts

```diff
@@ -54,6 +58,11 @@ export class MemPool {
     return this.transactions.size
   }
 
+  sizeBytes(): number {
+    const queueSize = this.size() * 32 // estimate the queue size
```

Comment:
Should this be `this.queue.size`?

Reply:
Yea I will change it. I thought it was a generator for some reasons and didn't have a size but it does